README.md

```diff
@@ -156,6 +156,43 @@ dropdb ert_db && createdb -O ert_user ert_db
 ert-storage alembic upgrade head
 ```
 
+### Database setup: Local setup
+
+First, download the required version of postgresql. For instance, for RHEL-7 you can check it [here](https://yum.postgresql.org/13/redhat/rhel-7-x86_64/repoview/).
+
+After installing or extracting the archive to a custom folder, append the postgres bin path to your actual PATH; eg. 
+
+```sh
+export PATH=$PATH:/data/install/postgres/usr/pgsql-13/bin
```

Comment:
Is this necessary? Is it possible to specify absolute paths when running the programs instead?

I'm thinking about:
1. User who knows what `PATH` is will be able to set it so they don't have to type a lot.
2. User who doesn't know what it does, and so telling them to execute commands with side-effects is not entirely desirable.

Reply:
I think I've tried the absolute path, but didn't work. I guess internally it runs another tools also.